test/validation/api/pktio/pktio.c

```diff
@@ -647,6 +594,49 @@ static int create_packets(odp_packet_t pkt_tbl[], uint32_t pkt_seq[], int num,
 				  true);
 }
 
+static void configure_pktio(int iface_idx, odp_pktio_t pktio, txrx_mode_flags_t flags,
+			    odp_pktin_mode_t imode, odp_pktout_mode_t omode,
+			    odp_schedule_sync_t sync_mode)
+{
+	odp_pktout_queue_param_t pktout_param;
+	odp_pktin_queue_param_t pktin_param;
+	odp_pktio_capability_t capa;
+
+	(void)(omode);
+
+	CU_ASSERT_FATAL(pktio != ODP_PKTIO_INVALID);
+	CU_ASSERT_FATAL(verify_txrx_mode_flags(flags) == 0);
+	CU_ASSERT_FATAL(odp_pktio_capability(pktio, &capa) == 0);
+
+	odp_pktin_queue_param_init(&pktin_param);
+	odp_pktout_queue_param_init(&pktout_param);
+
+	if (TXRX_FLAG_ISSET(flags, VECTOR) && capa.vector.supported) {
```

Comment:
The code should never end up here if vector mode is not supported, since vector tests are conditional.